### PR TITLE
Improve usability of Windows notes w/ examples for prompts & npm script

### PR DESCRIPTION
**Pull request by KyleStay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,22 +70,44 @@ Here are some examples:
...
+```cmd
+$env:DEBUG='app';node app.js
+```
+
 Then, run the program to be debugged as usual.
 
+npm script example:
+```js
+  "windowsDebug": "@powershell -Command $env:DEBUG='*';node app.js",
+
```
Comment (Qix-) [inline README.md]: Could you remove the extra line here?

File: README.md [hunk 2]
```diff
@@ -70,22 +70,44 @@ Here are some examples:
...
+```
+
```
Comment (Qix-) [inline README.md]: Could you remove the extra line here?

**Comment by Qix-** [comment]
Thanks again!

**Comment by KyleStay** [comment]
@Qix- , you are very welcome! This was my first pull request, thanks for helping me make it happen!

**Pull request merged**